components/org.wso2.carbon.identity.remotefetch.core/src/main/java/org/wso2/carbon/identity/remotefetch/core/dao/impl/RemoteFetchConfigurationDAOImpl.java

```diff
@@ -193,14 +180,15 @@ public void deleteRemoteFetchConfiguration(int configurationId) throws RemoteFet
     }
 
     /**
-     * @param tenantId
+     * @param tenantDomain
```

Comment:
Add a proper JavaDoc comment.

Reply:
Thank you, This comment and your previous comments were fixed in
https://github.com/wso2-extensions/identity-fetch-remote/pull/40
https://github.com/wso2-extensions/identity-fetch-remote/pull/41
https://github.com/wso2-extensions/identity-fetch-remote/pull/42
https://github.com/wso2-extensions/identity-fetch-remote/pull/43
https://github.com/wso2-extensions/identity-fetch-remote/pull/44
https://github.com/wso2-extensions/identity-fetch-remote/pull/45.
Hence I resolve this conversation.